examples/serve/min_replicas_zero.yaml

```diff
@@ -0,0 +1,23 @@
+# SkyServe YAML to test min_replicas=0 with a simple http server.
+#
+# Usage:
+#   sky serve up -n min_replicas examples/serve/min_replicas_zero.yaml
+# The endpoint will be printed in the console.
+# Querying the endpoint will trigger a scale up.
```

Comment:
Add docstr for how to make the replicas to 0, i.e. don;t send any traffic for how many time

Reply:
Thanks. updated